android/app/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 android {
     namespace = "com.nammaflutter.nammawallet"
     compileSdk = 36
-    ndkVersion = "28.2.13676358"
+    // ndkVersion = "28.2.13676358"
```

Comment:
This needs to be reverted. The version of NDK comes default with flutter will cause issue when using flutter_gemma. Pinning NDK version is safer that letting system decide which version to use.